tst/com/amazon/ion/benchmark/threshold.ion

```diff
@@ -0,0 +1,19 @@
+{input:"testStruct.10n",params:read::{format:"ION_BINARY",type:"FILE",api:"DOM"},relative_difference_score_threshold:{'·gc.alloc.rate':0.0000,'Heap usage':-0.01614003182541480555801658106555348482343250635890,'Serialized size':0.000000,speed:-0.0913183279742764692953173302096577494641544817982732}}
```

Comment:
So a negative score means that `new` is _slower_ than `previous`, right? Therefore, you check to see if the actual score is more negative than the threshold.

I recommend reducing the precision of the thresholds. There are very few measurements in the universe that require 50 digits of precision. Let's round to 4 digits.